backend/project_alpha/api/tests/test_views.py

```diff
@@ -117,3 +117,48 @@ def test_mock_if_except_password_validation_error(self):
             })
             self.assertIn('test', content['errors'][0]['message'])
             self.assertEqual(status_code, 400)
+
+class UserPreferencesAPIViewTestCase(APITestCase):
+
+    def setUp(self):
+        self.username = 'test@example.com'
+        self.pwd = '1234QWERty'
+        self.user = User.objects.create_user(self.username, self.pwd)
+
+    def test_happy_path_update_user_preferences(self):
+        self.client.force_authenticate(user=self.user)  # pylint: disable=no-member
+        response = self.client.patch(reverse('preferences'), data={'show_email': True,
+                                                                   'send_emails_with_news': True,
+                                                                   'timezone': 'Test/test',
+                                                                   'about_user': 'TEST',
+                                                                   'send_updates_threads': True,
+                                                                   'send_user_reviews': True,
+                                                                   'send_user_quests_reviews': True,
+                                                                   'send_updates_messages': True,
+                                                                   }, format='json')
+        usersettings = UserSettings.objects.get(user=self.user)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(usersettings.show_email)
+        self.assertTrue(usersettings.send_emails_with_news)
+        self.assertEqual(usersettings.timezone, 'Test/test')
+        self.assertEqual(usersettings.about_user, 'TEST')
+        self.assertTrue(usersettings.send_updates_threads)
+        self.assertTrue(usersettings.send_user_reviews)
+        self.assertTrue(usersettings.send_user_quests_reviews)
+        self.assertTrue(usersettings.send_updates_messages)
+
+    def test_happy_path_get_user_preferences(self):
+        self.client.force_authenticate(user=self.user)  # pylint: disable=no-member
+        response = self.client.get(reverse('preferences'), format='json')
+        usersettings = UserSettings.objects.get(user=self.user)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertFalse(usersettings.show_email)
+        self.assertFalse(usersettings.send_emails_with_news)
+        self.assertEqual(usersettings.timezone, 'UTC')
+        self.assertEqual(usersettings.about_user, '')
+        self.assertFalse(usersettings.send_updates_threads)
+        self.assertFalse(usersettings.send_user_reviews)
+        self.assertFalse(usersettings.send_user_quests_reviews)
+        self.assertFalse(usersettings.send_updates_messages)
```

Comment:
I think we're missing one more test case:data in request to update does not match serializer